packages/xstate-inspect/src/browser.ts

```diff
@@ -138,19 +139,24 @@ export function inspect(
     inspectService.send({ type: 'unload' });
   });
 
+  const stringifyWithSerializer = (value: any) =>
+    stringify(value, options?.serialize);
```

Comment:
nit: all other options are "unpacked" at the top of the outer function - maybe the same could be done with this to keep the style consistent?